modules/perception/lidar/lib/detection/lidar_point_pillars/point_pillars_detection.cc

```diff
@@ -43,6 +44,18 @@ bool PointPillarsDetection::Init(const DetectionInitOptions& options) {
   point_pillars_ptr_.reset(new PointPillars(
       FLAGS_reproduce_result_mode, FLAGS_score_threshold,
       FLAGS_nms_overlap_threshold, FLAGS_pfe_onnx_file, FLAGS_rpn_onnx_file));
+
+  // point cloud range
+  x_min_ = Params::kMinXRange;
```

Comment:
Try to initialize this x/y/z_min/max_ values from within header files or constructors, rather than inside the `Init` function.